src/main/java/TicTacToeCLI.java

```diff
@@ -0,0 +1,147 @@
+import java.util.Scanner;
+
+public class TicTacToeCLI {
+
+    public static void main(String[] args) {
+        TicTacToeCLI ticTacToe = new TicTacToeCLI();
+        System.out.println("Welcome to TicTacToe!");
+
+        Scanner in = new Scanner(System.in);
+
+        //initialize to class level.
+        //move the array from main.
+        // iterate
+        char[][] ticTacToeArray = {{'1', '2', '3'}, {'4', '5', '6'}, {'7', '8', '9'}};
+        ticTacToe.displayBoard(ticTacToeArray);
+
+
+//        System.out.println("Player1 - Please enter your name");
```

Comment:
Please don't commit commented lines of code to git.

Reply:
Done.